crates/scroll/node/src/pool.rs

```diff
@@ -0,0 +1,544 @@
+use alloy_consensus::{constants::EIP1559_TX_TYPE_ID, Transaction, Typed2718};
```

Comment:
Because of the changes to the `NodePromitives` for Scroll introduced in this PR, we can't use `NoopTransactionPool` anymore, because it sets the `GAT` for `TransactionPool` to [EthPooledTransaction](https://github.com/scroll-tech/reth/blob/64214fd1861d1367fe10432d4d300b407b0cc563/crates/transaction-pool/src/traits.rs#L1189), which uses `TransactionSigned` as a default generic type for the transaction